src/routes/routes.ts

```diff
@@ -19,6 +19,6 @@ router.get("/", renderTemplate(Templates.INDEX));
 
 router.get(pageURLs.COMPANY_NUMBER, renderTemplate(Templates.COMPANY_NUMBER));
 router.post(pageURLs.COMPANY_NUMBER, ...companyNumberRoute);
```

Comment:
Just wondering if we should (can) change that .post function above to not use the ellipses '...'?

Reply:
yes we can, i've updated it now